delivery_roulier_chronopost_fr/__manifest__.py

```diff
@@ -15,6 +15,11 @@
         "data/product.xml",
         "data/delivery.xml",
     ],
+    "external_dependencies": {
+        "python": [
+            "pytest-recording",
+        ],
+    },
```

Comment:
I'm not sure we want this dependency here.
It's only required for tests, not to use the module.
Normally we only need to update the `test-requirements.txt` file, I believe